familyconnect_registration/settings.py

```diff
@@ -243,3 +250,19 @@
     'VHT_PUBLIC_REGISTRATION_NOTIFICATION_TEXT', "There is a new pregnancy in "
     "your parish. Call {mother} and visit the mother to update her "
     "registration.")
+LOCATION_UPDATE_REMINDER_TEXT_ENG_UG = os.environ.get(
+    'LOCATION_UPDATE_REMINDER_TEXT_ENG_UG', 'To make sure you can receive '
```

Comment:
Yes, this is how I'll be handling translations. `eng_UG` is how the jsbox app stores them https://github.com/praekelt/chms-jsbox/blob/develop/src/ussd_public.js#L250-L253 , so that's why I've got it like that.

I have no idea why the ones above are `UG_ENG`, but by looking at the code, it doesn't look like those ones handle translations, so not sure why the language code is there..